### ISSUE TITLE
Request: Traction

### ISSUE THREAD

**Issue by HighSeasCaeser** [cam72cam/ImmersiveRailroading#61]
Add the ability for locomotives to use sand to gain additional traction when pulling heavy loads.

**Comment by Thomas4peyton** [comment]
I agree, sanders would be a great addition; I don't know how they would work but most likely you would get lets say, 5,000lbs extra tractive effort while the sanders are on: which lets say you could hold 2-4 stacks of sand blocks in the locomotive to use for sanding.
